src/Bridge/Doctrine/Orm/Filter/NumericFilter.php

```diff
@@ -0,0 +1,148 @@
+     *
+     * For each property passed, if the resource does not have such property or if the  value is not numerical the property is ignored.
+     */
+    public function apply(QueryBuilder $queryBuilder, string $resourceClass, string $operationName = null)
+    {
+        $request = $this->requestStack->getCurrentRequest();
+        if (null === $request) {
+            return;
+        }
+
+        $properties = $this->extractProperties($request);
+
+        foreach ($properties as $property => $numerical) {
+            if (!$this->isPropertyEnabled($property) || !$this->isPropertyMapped($property, $resourceClass)) {
+                continue;
+            }
+
+            if (empty($numerical) && isset($this->properties[$property])) {
+                $numerical = $this->properties[$property];
+            }
+            if (!is_numeric($numerical) && !empty($numerical)) {
+                continue;
+            }
+
+            $alias = 'o';
+            $field = $property;
+
+            if ($this->isPropertyNested($property)) {
+                $propertyParts = $this->splitPropertyParts($property);
+
+                $parentAlias = $alias;
+
+                foreach ($propertyParts['associations'] as $association) {
+                    $alias = QueryNameGenerator::generateJoinAlias($association);
+                    $queryBuilder->leftJoin(sprintf('%s.%s', $parentAlias, $association), $alias);
+                    $parentAlias = $alias;
+                }
+
+                $field = $propertyParts['field'];
+            }
+            $valueParameter = QueryNameGenerator::generateParameterName($field);
+
+            $queryBuilder
+                ->andWhere(sprintf('%s.%s = :%s', $alias, $field, $valueParameter))
+                ->setParameter($valueParameter, $numerical);
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDescription(string $resourceClass) : array
+    {
+        $description = [];
+
+        $properties = $this->properties;
+        if (null === $properties) {
+            $properties = array_fill_keys($this->getClassMetadata($resourceClass)->getFieldNames(), null);
+        }
+
+        foreach ($properties as $property => $numerical) {
+            if (!$this->isPropertyMapped($property, $resourceClass) || !$this->isNumericalField($property, $resourceClass)) {
+                continue;
+            }
+            $propertyParts = $this->splitPropertyParts($property);
+            $metadata = $this->getNestedMetadata($resourceClass, $propertyParts['associations']);
+
+            $description[$property] = [
+                'property' => $property,
+                'type' => $metadata->getTypeOfField($propertyParts['field']),
+                'required' => false,
+            ];
+        }
+
+        return $description;
+    }
+
+    /**
+     * Determines whether the given property is a numerical or not.
```

Comment:
What I'm saying is that we should stick to "numeric", because that's what PHP uses (e.g. `is_numeric`).